### PR TITLE
Forward wizard mode through nle, add tests.

### PR DESCRIPTION
**Pull request by yobibyte**

### CODE REVIEW

File: nle/tests/test_envs.py [hunk 1]
```diff
@@ -75,25 +75,26 @@ def compare_rollouts(env0, env1, max_rollout_len):
 
 
 @pytest.mark.parametrize("env_name", get_nethack_env_ids())
+@pytest.mark.parametrize("wizard", [False, True])
```
Comment (heiner) [inline nle/tests/test_envs.py:78]: Thanks for the elaborate test!

I'm a little bit reluctant though to double the runtime of the test like this, although obviously it gives optimal test coverage this way.  Perhaps we could not add `wizard` to `TestGymEnvRollout`?

**Review by heiner — APPROVED** [review]
Thanks!

Small question about the test below.

File: nle/tests/test_envs.py (hunk 1 above)
Comment (yobibyte) [reply nle/tests/test_envs.py:78]: I was not sure what wizard-mode affects under the hood, that's why I added the option. I can remove not to increase the testing time.

**Review by rockt — APPROVED** [review]

**Review by heiner — APPROVED** [review]
Thanks! Looks good.

**Pull request merged**